database/dataGenerator.js

```diff
@@ -0,0 +1,65 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const faker = require('faker');
+// const db = require('./index.js');
+
+
+const dateGenerator = () => {
```

Comment:
you might consider using faker's own date methods for generating random dates.
https://github.com/Marak/faker.js/wiki/Dates
though the date returned from faker also contains a timestamp, since your db table schema is using DATEONLY for the date field, it will be formatted correctly when it is seeded.